generator/java.stoneg.py

```diff
@@ -2011,8 +2010,12 @@ def builder_class(self, stone_elem):
 
         if isinstance(stone_elem, ApiRoute):
             route = stone_elem
+            # Use prefix here because multiple builders may be generated
+            # if the endpoint has multiple auth types
+            prefix = self._args.requests_classname_prefix or self._args.client_class
             package = self.java_class(route).package
-            return JavaClass(package + '.' + classname(format_func_name(route) + '_builder'))
+            return JavaClass(package + '.' + classname(
```

Comment:
I think this is backward-incompatible. Like `GetMetadataBuilder` will become `DbxUserGetMetadataBuilder`.  It would be great if we add prefix only for routes with multiple auth

Reply:
Good idea, updated